nuxt/src/runtime/plugin.ts

```diff
@@ -1,9 +1,11 @@
+import { AForm } from '@stonecrop/aform'
 import { Stonecrop } from '@stonecrop/stonecrop'
 import { useRouter } from 'nuxt/app'
 
 import { defineNuxtPlugin } from '#app'
 
 export default defineNuxtPlugin(_nuxtApp => {
 	const app = _nuxtApp.vueApp
+	app.component('AForm', AForm)
```

Comment:
This should probably use the plugin pattern to make all components available rather adding the components one-by-one.